docs/application/native/guides/connectivity/softap.md

```diff
@@ -0,0 +1,341 @@
+# SoftAP
+
+SoftAP means Software Access Point for local area connections without internet access.
+
+The main features of the SoftAP API include:
+
+- Enabling and Disabling SoftAP
+
+ You can turn SoftAP on or off.
+
+
+- Managing SoftAP
+
+ You can manage SoftAP configuration settings, such as the Service Set Identifier (SSID) and passphrase.
+The following table lists the attributes of SoftAP.
+
+**Table: the attributes of SoftAP**
+<table>
+    <thead>
+	    <tr>
+		   <th>Attributes</th>
+		   <th>Description</th>
+		</tr>
+	</thead>
+	<tbody>
+	    <tr>
+            <td>SSID</td>
+			<td>SSID (Service set identifier)</td>
+		</tr>
+	    <tr>
+            <td>Security type</td>
+			<td>None, WPA2_PSK and WPS</td>
+		</tr>
+	    <tr>
+            <td>Passphrase</td>
+			<td>Passphrase for wireless access</td>
+		</tr>
+	    <tr>
+            <td>Visibility</td>
+			<td>Visibility of SSID</td>
+		</tr>
+	    <tr>
+            <td>IP address</td>
+			<td>Local IP address</td>
+		</tr>
+	    <tr>
+            <td>Mode</td>
+			<td>Wireless mode (B, G, A and AD)</td>
+		</tr>
+	    <tr>
+            <td>Channel</td>
+			<td>Channel</td>
+		</tr>
+	</tbody>
+</table>
+
+- SoftAP Client
+
+ You can get information about connected client such as name, IP address, MAC address and connection time.
+
+
+## Prerequisites
+
+1. To use the SoftAP API (in [mobile](../../api/mobile/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) applications), the application has to request permission by adding the following privileges to the 'tizen-manifest.xml' file:
```

Comment:
To use the SoftAP API (in [mobile](../../api/mobile/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) applications), the application must request permission by adding the following privileges to the 'tizen-manifest.xml' file: